test/unit/specs/Slides.spec.js

```diff
@@ -2,10 +2,55 @@ import Vue from 'vue'
 import Slides from '@/components/Slides'
```

Comment:
In this file is where we'd have a mock of slides instead of having it hardcoded

Reply:
Is this one related to the comment on line 59 of the same file ?

Reply:
If I remove the hardcoded stuff, there won't be anything to view in the saved slides, is that fine?

Reply:
This will reflect on the test cases too, because the test cases are using this dummy data to test the functionality of the component

Reply:
You'll need to create mocks of the slide data

Reply:
I've made some changes....